src/components/ActionControls/HighlightControls.js

```diff
@@ -9,29 +10,34 @@ import './HighlightControls.scss';
 
 type Props = {
     canDelete: boolean,
-    canAnnotate: boolean,
+    canComment: boolean,
+    isPending?: boolean,
     onCreate: Function,
     onCommentClick: Function
 };
 
-const HighlightControls = ({ canDelete, canAnnotate, onCreate, onCommentClick }: Props) => (
+const HighlightControls = ({ canDelete, canComment, onCreate, onCommentClick, isPending }: Props) => (
     <div className='ba-action-controls-highlight'>
         {canDelete && (
-            <PlainButton type='button' className='ba-highlight-btn' onClick={onCreate}>
-                <IconHighlightAnnotation />
+            <PlainButton type='button' className='ba-highlight-btn' onClick={onCreate} isDisabled={isPending}>
```

Comment:
It seems strange that `canDelete` is the condition on which we show the create highlight button. Not sure if it's a matter of just changing the prop name but it doesn't seem like delete is in the scope of this component?

Reply:
I can rename the prop. The button needs canDelete because it also serves as the delete button when the annotation has been saved.